### PR TITLE
Add Exoscale to the CSI drivers list

### PR DESCRIPTION
**Pull request by pierre-emmanuelJ**
Add the Exoscale driver to the list of CSI drivers.

### CODE REVIEW

File: book/src/drivers.md [hunk 1]
```diff
@@ -48,6 +48,7 @@ Name | CSI Driver Name | Compatible with CSI Version(s) | Description | Persiste
...
 [Ember CSI](https://ember-csi.io) | `[x].ember-csi.io` | v0.2, v0.3, v1.0 | Multi-vendor CSI plugin supporting over 80 Drivers to provide block and mount storage to Container Orchestration systems. | Persistent | Read/Write Single Pod | Yes | Raw Block, Snapshot
 [Excelero NVMesh](https://github.com/Excelero/nvmesh-csi-driver) | `nvmesh-csi.excelero.com` | v1.0, v1.1 | A Container Storage Interface (CSI) Driver for Excelero NVMesh | Persistent | Read/Write Multiple Pods | Yes | Raw Block, Expansion
+[Exoscale CSI](https://github.com/exoscale/exoscale-csi-driver) | `csi.exoscale.com` | v1.x | A Container Storage Interface (CSI) Driver for Exoscale Block Storage | Persistent | Read/Write Single Pod | Yes | Raw Block, Snapshot, Topology
```
Comment (xing-yang) [inline book/src/drivers.md:51]: Please change v1.x to a CSI spec version that your driver supports.

**Comment by xing-yang** [comment]
Please squash the commits.

File: book/src/drivers.md (hunk 1 above)
Comment (xing-yang) [reply book/src/drivers.md:51]: I see here it is v1.8: https://github.com/exoscale/exoscale-csi-driver/blob/main/go.mod#L6

Comment (pierre-emmanuelJ) [reply book/src/drivers.md]: https://github.com/kubernetes-csi/docs/pull/586/commits/b3e95f73f46ba1fe04d6e982f62a9aebdfe0b697

**Pull request merged**